.github/workflows/tests-ci.yml

```diff
@@ -82,7 +82,7 @@ jobs:
       matrix:
         include:
           - group: "A"
-            packages: "--workspace packages/agent --workspace packages/dids "
+            packages: "--workspace packages/agent --workspace packages/dids"
```

Comment:
need to add the new package here?

Reply:
@nitro-neal Not for that set of tests.  The new `@web5/crypto-aws-kms` package isn't bundled for nor usable in web browsers.